sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubConsumerClient.java

```diff
@@ -119,41 +122,46 @@ public EventHubProperties getProperties() {
      *
      * @param partitionId The unique identifier of a partition associated with the Event Hub.
+     *
      * @return The set of information for the requested partition under the Event Hub this client is associated with.
      */
     public PartitionProperties getPartitionProperties(String partitionId) {
         return consumer.getPartitionProperties(partitionId).block(timeout);
     }
 
     /**
-     * Receives a batch of EventData from the Event Hub partition.
+     * Receives a batch of {@link PartitionEvent events} from the Event Hub partition.
      *
      * @param maximumMessageCount The maximum number of messages to receive in this batch.
      * @param partitionId Identifier of the partition to read events from.
-     * @return A set of {@link EventData} that was received. The iterable contains up to {@code maximumMessageCount}
-     *     events. If a stream for the events was opened before, the same position within that partition is returned.
-     *     Otherwise, events are read starting from {@link #getStartingPosition()}.
+     *
+     * @return A set of {@link PartitionEvent} that was received. The iterable contains up to
+     *     {@code maximumMessageCount} events. If a stream for the events was opened before, the same position within
+     *     that partition is returned. Otherwise, events are read starting from {@link #getStartingPosition()}.
+     *
      * @throws IllegalArgumentException if {@code maximumMessageCount} is less than 1.
      */
     public IterableStream<PartitionEvent> receive(String partitionId, int maximumMessageCount) {
         return receive(partitionId, maximumMessageCount, timeout);
     }
 
     /**
-     * Receives a batch of EventData from the Event Hub partition
+     * Receives a batch of {@link PartitionEvent events} from the Event Hub partition.
      *
      * @param partitionId Identifier of the partition to read events from.
      * @param maximumMessageCount The maximum number of messages to receive in this batch.
      * @param maximumWaitTime The maximum amount of time to wait to build up the requested message count for the
      *     batch; if not specified, the default wait time specified when the consumer was created will be used.
-     * @return A set of {@link EventData} that was received. The iterable contains up to {@code maximumMessageCount}
-     *     events.
+     *
+     * @return A set of {@link PartitionEvent} that was received. The iterable contains up to
+     *     {@code maximumMessageCount} events.
+     *
      * @throws NullPointerException if {@code maximumWaitTime} is null.
      * @throws IllegalArgumentException if {@code maximumMessageCount} is less than 1 or {@code maximumWaitTime} is
      *     zero or a negative duration.
      */
     public IterableStream<PartitionEvent> receive(String partitionId, int maximumMessageCount,
-            Duration maximumWaitTime) {
+        Duration maximumWaitTime) {
```

Comment:
.NET uses `ReadFromPartitionAsync`. While I don't like `read`, should we use `receiveFromPartition` to be closer to what .NET is using?

Reply:
I'm going to defer these naming changes until we've finished our review because it results in a lot of changed code. I'll create an issue for it though.